### DISCUSSION TITLE
About preventing the creation of duplicate IP addresses

### DISCUSSION THREAD

**Discussion by wypxl** [netbox-community/netbox#13831] [General]
Hi，guys
  My NetBox version is 3.6.0, my python version is 3.9.5.

  I found that I could create duplicate IP addresses in the WEB interface, although conflict relationships were displayed on the WEB  interface after creating. Is there a way to show conflicting relationships at creation time and then fail to create it.

**Comment by candlerb** [comment]
The setting you want is called [ENFORCE_GLOBAL_UNIQUE](https://docs.netbox.dev/en/stable/configuration/miscellaneous/#enforce_global_unique).

You can set it in `configuration.py` or via "Admin > Config revisions" in the web interface (but note there was a bug in 3.6.0 if you don't have any config revisions created yet, so upgrade to 3.6.1 first if you want to do it that way)

**Reply by wypxl** [reply]
OK, i followed the first way adn have solved that, thanks a lot.
